Tools/GNUMake/comps/intel.mak

```diff
@@ -37,11 +37,13 @@ endif
 
 ########################################################################
 
-ifeq ($(firstword $(sort 17.0 $(intel_version))), 17.0) 
-  CXXFLAGS += -std=c++14
-else
-  CXXFLAGS += -std=c++11
+ifneq ($(firstword $(sort 17.0 $(intel_version))), 17.0)
+  ifeq ($(shell expr $(CXXSTD) \> 11),1)
+    $(error C++14 support requires Intel icpc 17.0 or newer.)
+  endif
```

Comment:
It seems that for Intel >= 18 it now uses C++11 by default.

Reply:
yes, as above.

Reply:
updated to old behaviour (which is also closer to CMake's behaviour)